packages/@lwc/shared/src/api-version.ts

```diff
@@ -10,15 +10,20 @@ import { isNumber } from './language';
 export const enum APIVersion {
     V58_244_SUMMER_23 = 58,
     V59_246_WINTER_24 = 59,
+    V60_248_SPRING_24 = 60,
```

Comment:
Minor nit: Since we're going to be seeing more of these, it'd be nice to have them introduced in a separate PR to keep commit history clean & PRs more readable.

Reply:
Fair enough. I avoided it since it was like 3 lines though, and since I knew I would have to deal with conflicts anyway once I merged everything all together.